doc/OPTIONS

```diff
@@ -448,7 +448,7 @@ separate invocations of John, one per hash type.
 
 For performance reasons, Jumbo preloads the wordlist file into memory instead
 of reading line by line, when rules are in use.  The default max. size file
-for this to happen is shown in --list=hidden-options.  A special value is
+for this to happen is shown in among the --help options.  A special value is
```

Comment:
The default max. size file for this to happen is shown ~**in**~ among the --help options.

Reply:
Thanks, just fixed.